u2f-chrome-extension/approvedorigins.js

```diff
@@ -1,13 +1,8 @@
-// Copyright 2014 Google Inc. All rights reserved
-//
-// Use of this source code is governed by a BSD-style
-// license that can be found in the LICENSE file or at
-// https://developers.google.com/open-source/licenses/bsd
-
 /**
  * @fileoverview Provides an interface to check whether the user has approved
  * an origin to use security keys.
  *
+ * @author juanlang@google.com (Juan Lang)
```

Comment:
I think these should have been removed by the export tool? I'm okay with my name being on these files, but it seems like the export might not have worked. See below.